fast/stages-multitenant/0-bootstrap-tenant/organization.tf

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Organization tag and conditional IAM grant.
+
+locals {
+  iam_tenant_condition = "resource.matchTag('${local.tag_keys.tenant}', '${var.tenant_config.short_name}')"
+  tag_keys = {
+    for k, v in var.tag_names : k => "${var.organization.id}/${v}"
+  }
+}
+
+module "organization" {
+  source          = "../../../modules/organization"
+  organization_id = "organizations/${var.organization.id}"
+  iam_additive = merge(
+    {
+      "roles/resourcemanager.organizationViewer" = [
+        "group:${local.groups.gcp-admins}"
+      ]
+    },
+    var.billing_account.is_org_level ? {
+      "roles/billing.admin" = [
+        "group:${local.groups.gcp-admins}",
+        module.automation-tf-resman-sa.iam_email
+      ]
+      "roles/billing.costsManager" = ["group:${local.groups.gcp-admins}"]
+    } : {}
+  )
+  tags = {
```

Comment:
I might be missing something but I think you're ignoring all the other tags in `var.tag_values` and `var.tag_keys`

Reply:
yep we are, those are produced for consumption by everything that comes after org resman, here we only need tenant